src/openrct2/paint/sprite/Paint.Misc.cpp

```diff
@@ -134,7 +134,9 @@ void misc_paint(paint_session* session, const SpriteBase* misc, int32_t imageDir
                 isAntiClockwise = !isAntiClockwise;
             }
 
-            uint32_t baseImageId = (jumpingFountain->type == SPRITE_MISC_JUMPING_FOUNTAIN_SNOW) ? 23037 : 22973;
+            uint32_t baseImageId = (static_cast<MiscEntityType>(jumpingFountain->type) == MiscEntityType::JumpingFountainSnow)
+                ? 23037
```

Comment:
These naked numbers should get a constant. Like `SPR_JUMPING_FOUNTAIN_WATER_START` and `SPR_JUMPING_FOUNTAIN_SNOW_START`.

Reply:
I'll do that in another PR, there are more naked numbers in this file like that.